builtin/logical/ssh/backend_test.go

```diff
@@ -149,7 +149,7 @@ func prepareTestContainer(t *testing.T, tag, caPublicKeyPEM string) (func(), str
 	sshAddress := fmt.Sprintf("127.0.0.1:%s", resource.GetPort("2222/tcp"))
 
 	// exponential backoff-retry
-	pool.MaxWait = 10 * time.Second
+	//pool.MaxWait = 10 * time.Second
```

Comment:
why leave this commented out and not just remove it entirely?

Reply:
Eh, when doing local development the default maxwait is annoyingly high, and I figured I'd save someone from having to lookup how to fix it.  Probably wrong-headed, I'll remove it.

Reply:
Gotcha, maybe just leave a comment instead?